### PR TITLE
Update Sandbox API key procedure (bunq/sdk_csharp#101)

### PR DESCRIPTION
**Pull request by sandervdo**
[//]: # (Thanks for opening this pull request! Before you proceed please make sure that you have an issue that explains what this pull request will do.
         Make sure that all your commits link to this issue e.g. "My commit. \(bunq/sdk_csharp#<issue nr>\)".
         If this pull request is changing files that are located in "BunqSdk/Model/Generated" then this pull request will be closed as these files must/can only be changed on bunq's side.)
         
## This PR closes/fixes the following issues:
[//]: # (If for some reason your pull request does not require a test case you can just mark this box as checked and explain why it does not require a test case.)
 - Closes bunq/sdk_csharp#101
    - [x] Tested -> not necessary. Doc change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ Now you can build even bigger and better apps and integrate them with your bank
 
 Before you dive into this brand new SDK, please consider:
 - Checking out our new developer’s page [https://bunq.com/en/developer](https://bunq.com/en/developer) 🙌  
-- Grabbing your production API key from the bunq app or asking our support for a Sandbox API key 🗝
+- Grabbing your production API key from the bunq app or generate a Sandbox API key using the
+[Tinker endpoint](https://together.bunq.com/topic/tinker-the-api) 🗝
```
Comment (OGKevin) [inline README.md]: Same, `...key using tinker`

**Comment by sandervdo** [comment]
@OGKevin pushed.

**Comment by OGKevin** [comment]
@sandervdo

**Comment by sandervdo** [comment]
@OGKevin didn't you mean to tag Andre or sth? As for the period, isn't that a bit of a waste of characters? There's a limit on the amount of characters that are shown on GitHub anyway, the remainder is cut off to the description part (that's why I missed it I guess) 😋

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**